screens/Login.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, Button, Alert } from 'react-native';
+import auth from '@react-native-firebase/auth';
+import { globalStyles } from '../styles/global';
+import { Formik } from 'formik';
+import { TextInput } from 'react-native-gesture-handler';
+import Home from './Home';
+import * as yup from 'yup';
+
+const loginSchema = yup.object({
+  email: yup
+    .string()
+    .email()
+    .required(),
+  password: yup
+    .string()
+    .required()
+});
+
+export default function Login({ navigation }) {
+
+  // Set an initializing state whilst Firebase connects
+  const [initializing, setInitializing] = useState(true);
+  const [user, setUser] = useState();
+
+  redirect = () => {
+    navigation.navigate("Signup")
+  }
+
+  //Signs the user in
+  login = (email, password) => {
+    auth()
+    .signInWithEmailAndPassword(email, password)
+    .then(() => {
+        console.log('User signed in successfully!');
+    })
+    .catch(error => {
+      switch(error.code) {
+        case 'auth/invalid-email':
+          Alert.alert('Invalid Email Format Submitted')
+          break;
+        case 'auth/wrong-password':
+          Alert.alert('Incorrect Email or Password Submitted')
+          break;
+        case 'auth/user-not-found':
+          Alert.alert('Incorrect Email or Password Submitted')
+          break;
+      }
+    });
+  }
+
+  // Signs the user out
+  signOut = () => {
```

Comment:
I'm not sure if you tested this or if VS code just isn't showing it as an error, but since this function is inside a functional component (i.e. not a class component), I believe it would need to be defined. 

```suggestion
  const signOut = () => {
```

Arrow functions `() => {}` are just a way of creating a function without a name, so when you specify `signOut = () => {}` you are assigning the `signOut` variable to an arrow function, but the variable should still needs to be declared and since this function will never change it is reasonable to assign it to a `const`

Reply:
I don't recall VS Code displaying any error messages despite the function being inside a functional component, however what you added at the end of your comment there makes perfect sense. It's reasonable to assign it to a const since I declared a signOut variable. Thanks for the tidbit on arrow functions as well, helped me understand it just a bit better 😁